jwt/jwt/jwt.h

```diff
@@ -14,7 +14,7 @@
 #include "zowetypes.h"
 #include "json.h"
 #include "utils.h"
-#include "rs_icsfp11.h"
+#include "../rscrypto/rs_icsfp11.h"
```

Comment:
What about ```jwt/rscrypto/rs_icsfp11.h```? With the solution you have you may include something by accident (of course with the name rs_icsfp11.h it's unlikely), i.e. the compiler will scan something you didn't intend it to. Plus, you clearly know this is from jwt.

Reply:
Or alternatively, ```rscrypto/rs_icsfp11.h``` if rscrypto is a library on its own.

Reply:
This is for ZSS plugins that are unaware of this change and therefore can't (or don't want to) change their include path in the build script.

The plugin includes `httpserver.h`
`httpeserver.h` includes `jwt/jwt/jwt.h`
`jwt.h` includes `../rscrypto/rs_icsfp11.h`: this won't break the plugin 

`rscrypto/rs_icsfp11.h` won't work for that reason.

`jwt/rscrypto/rs_icsfp11.h` can also work,  but I don't think it's any better.  What can be at `jwt/jwt/../rscrypto/rs_icsfp11.h` "by accident" other than what's needed?

Reply:
> Do you think it will try to include ../rscrypto/rs_icsfp11.h from every folder on the -I list?
Possibly. I guess it's not a big issue with such unique names.

Anyway, if that's the best option, lgtm.